notification-portlet-webcomponents/notification-icon/package-lock.json

```diff
@@ -1760,7 +1760,7 @@
     },
     "acorn-jsx": {
       "version": "3.0.1",
-      "resolved": "https://registry.npmjs.org/acorn-jsx/-/acorn-jsx-3.0.1.tgz",
+      "resolved": "http://registry.npmjs.org/acorn-jsx/-/acorn-jsx-3.0.1.tgz",
```

Comment:
Making registry links http only? :confused:

Reply:
Hmm, I don't edit those files. Wonder how it changed to `http`?

Reply:
It probably changed when `npm install` was executed. I haven't dug into why this is, but I've seen this happen ...

Reply:
I'll revert these to `https` soon